README.md

```diff
@@ -1,2 +1,407 @@
-# blink-debit-api-client-java8
-The Blink Debit API client for Java 8
+# Blink Debit API Client for Java
+![badge](https://github.com/BlinkPay/Blink-Debit-API-Client-Java/actions/workflows/workflow.yml/badge.svg)
+
+This SDK allows merchants with Java-based e-commerce site to integrate with Blink PayNow and Blink AutoPay.
+
+# Minimum Requirements
+- Maven 3 or Gradle 7
+- Java 8
+- Lombok 1.18
+
+This SDK internally uses WebClient, a reactive web client introduced in Spring Framework 5, for making API calls.
+
+# Dependency
+## Maven
+```xml
+<dependency>
+    <groupId>nz.co.blinkpay</groupId>
+    <artifactId>blink-debit-api-client-java</artifactId>
+    <version>1.0.0</version>
+</dependency>
+```
+## Gradle
+```
+implementation 'nz.co.blinkpay:blink-debit-api-client-java:1.0.0
+```
+
```

Comment:
Let's include a full worked example here of a single consent using the most basic code set-up here.
(i.e. a full example of a quick payment)